uint16 live
uint16 min
uint16 max
